### PR TITLE
BancorVortex contract

### PR DESCRIPTION
**Pull request by barakman**

### CODE REVIEW

File: deploy/scripts/000020-set-network-settings.ts [hunk 1]
```diff
@@ -34,13 +34,6 @@ const func: DeployFunction = async ({ getNamedAccounts }: HardhatRuntimeEnvironm
         from: deployer
     });
 
-    await execute({
-        name: InstanceName.NetworkSettings,
-        methodName: 'setVortexRewards',
-        args: [{ burnRewardPPM: toPPM(10), burnRewardMaxAmount: toWei(100) }],
-        from: deployer
-    });
-
```
Comment (barakman) [inline deploy/scripts/000020-set-network-settings.ts:43]: TODO:
1. Reinstate
2. Rename to V1
3. Add a deployment script for the new version
4. Add a deployment test for the new version
5. Add a deployment script for the BancorVortex contract
6. Add a deployment test for the BancorVortex contract
7. Import the BancorVortex artifact in `/utils/Deploy.ts`

Comment (barakman) [inline deploy/scripts/000020-set-network-settings.ts:43]: TODO:
1. Reinstate
2. Rename to V1
3. Add a deployment script for the new version
4. Add a deployment test for the new version
5. Add a deployment script for the BancorVortex contract
6. Add a deployment test for the BancorVortex contract
7. Import the BancorVortex artifact in `/utils/Deploy.ts`

File: deploy/tests/000020-set-network-settings.ts [hunk 2]
```diff
@@ -16,9 +16,5 @@ describeDeployment(__filename, () => {
         expect(await networkSettings.networkFeePPM()).to.equal(toPPM(15));
         expect(await networkSettings.withdrawalFeePPM()).to.equal(toPPM(0.25));
         expect(await networkSettings.flashLoanFeePPM()).to.equal(toPPM(0.09));
-
-        const vortexRewards = await networkSettings.vortexRewards();
-        expect(vortexRewards.burnRewardPPM).to.equal(toPPM(10));
-        expect(vortexRewards.burnRewardMaxAmount).to.equal(toWei(100));
```
Comment (barakman) [inline deploy/tests/000020-set-network-settings.ts:22]: TODO: reinstate...

Comment (spartacus2690) [reply deploy/tests/000020-set-network-settings.ts:22]: __

**Pull request closed without merging**